src/TestInfrastructure/VsObjectModelMocks/ProjectMock.cs

```diff
@@ -24,16 +24,14 @@
 using System.Linq;
 using EnvDTE;
 using Microsoft.VisualStudio;
-using Microsoft.VisualStudio.Shell.Flavor;
 using Microsoft.VisualStudio.Shell.Interop;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests
 {
-    public class ProjectMock : VsUIHierarchyMock, IVsProject, Project, IVsBuildPropertyStorage, IVsAggregatableProjectCorrected
+    public class ProjectMock : VsUIHierarchyMock, IVsProject, Project, IVsBuildPropertyStorage
```

Comment:
would be nice to say that this a 'new' project mock because it does not have `IVsAggregatableProjectCorrected` ?